cloud/services/virtualmachines/virtualmachines.go

```diff
@@ -158,10 +167,41 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 		return errors.Wrapf(err, "cannot create vm")
```

Comment:
```suggestion
		return errors.Wrapf(err, "cannot create VM")
```

my bad, I missed that one in #544

Reply:
will go through all of them